clubManagement/views.py

```diff
@@ -68,3 +68,80 @@ def get(self, request, *args, **kwargs):
                 month_att.append(len(att.filter(date__month=(i + 1))))
             context = {'user': user, 'month_att': month_att, 'month': month, 'month_num': month_num, 'year': kwargs.get('year')}
         return render(request, self.template_name, context)
+
+
+class MonthAttendanceReportView(View):
```

Comment:
This function should take the batch from the url, and filter UserInfo. Loop through that result and save the user whose Attendance matches month and year. 

The final result should be a list containing {user, attendance for that month}. loop through this list in the html and show each students attendance for that month.

Ex: show attendance for all 1st years, for 2017/5, /club/attendance-report/2016/2017/05/